src/scripts/Save.ts

```diff
@@ -0,0 +1,35 @@
+class Save {
+
+    static counter: number = 0;
+
+    public static store(player: Player) {
+        let json = JSON.stringify(player);
+        localStorage.setItem("player", json);
+        this.counter = 0;
+    }
+
+    public static load(): Player {
+        let saved = localStorage.getItem("player");
+        if (saved) {
+            return new Player(JSON.parse(saved));
+        } else {
+            return new Player()
+        }
+    }
+
+    public static reset(): void {
+        localStorage.setItem("player", null);
+        location.reload()
+    }
+
+    public static filter(object: any, keep: string[]): Object {
+        let filtered = {}, prop;
+        for (prop in object) {
+            if (keep.indexOf(prop) > -1) {
+                filtered[prop] = object[prop]
+            }
+        }
+        return filtered
+    }
+
```

Comment:
Should I add a comment explaining the parameters of that function, or is it fairly clear what they should be?